### DISCUSSION TITLE
Idea:  It would be nice for this module to have an updated crate

### DISCUSSION THREAD

**Discussion by peepo5** [fabianboesiger/ftx#76] [General]
I am trying to package my app for cargo to upload onto crates.io, however it will not allow me to do it because my ftx dependency is relying on the nightly version of this module, which does not have a version number. A temporary fix could be upping the version release again inside github because passing 
`ftx = { git="https://github.com/fabianboesiger/ftx", branch = "main", version = "0.4" }` made it work, but then the latest fixes caused it to not work properly. It would be nice to have a release schedule for cargo, or for the package version to be updated more frequently, at least inside this repo.

**Comment by fabianboesiger** [comment]
I'll try to update the crates.io version more frequently from now on. But with daily PRs I can't guarantee that the crates.io version always is up to date.

**Reply by peepo5** [reply]
alright. could you change the gh vers as in my pull request, thanks.

**Reply by peepo5** [reply]
oh nvm i see its updated now thank you :)
